.github/workflows/label.yml

```
@@ -32,7 +32,7 @@ jobs:
 
     # Label based on modified files
     - name: Label based on changed files
-      uses: actions/labeler@v4
+      uses: actions/labeler@v5
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         sync-labels: ''
```

```suggestion
        sync-labels: false
```
see https://github.com/actions/labeler/issues/112